devito/propagator.py

```diff
@@ -425,7 +425,9 @@ def sympy_to_cgen(self, stencils):
                 sub = str(ccode(self.time_substitutions(expr).xreplace(self._mapper)))
                 if self.dtype is np.float32:
                     factors.append(cgen.Assign(name, (sub.replace("pow", "powf")
-                                                      .replace("fabs", "fabsf"))))
+                                                      .replace("fabs", "fabsf")
+                                                      .replace("cos", "cosf")
+                                                      .replace("sin", "sinf"))))
```

Comment:
Not to be fixed as part of this PR, but these replacements should really live in the `CodePrinter`.